weather/ngsild-payloads/weatherObserved-multiple-sources

```diff
@@ -0,0 +1,248 @@
+{
+  "id": "urn:ngsi-ld:WeatherObserved:Italy:Calabria:Lappano",
+  "type": "WeatherObserved",
+  "precipitationIntensity": {
+    "type": "Property",
+    "datasetId": "urn:ngsi-ld:Dataset:WeatherStation",
+    "observedBy": {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:Device:WeatherStation:dc94e9f5-6e3a-443d-88b4-778d2519b3c6"
+    },
+    "value": 12,
+    "observedAt": "2022-01-05T23:00:00Z",
+    "unitCode": "E92"
+  },
+  "relativeHumidity80": {
+    "type": "Property",
+    "datasetId": "urn:ngsi-ld:Dataset:WeatherStation",
+    "observedBy": {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:Device:WeatherStation:dc94e9f5-6e3a-443d-88b4-778d2519b3c6"
+    },
+    "value": 180,
+    "observedAt": "2022-01-05T23:00:00Z",
+    "unitCode": "SEC"
+  },
+  "relativeHumidity90": {
+    "type": "Property",
+    "datasetId": "urn:ngsi-ld:Dataset:WeatherStation",
+    "observedBy": {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:Device:WeatherStation:dc94e9f5-6e3a-443d-88b4-778d2519b3c6"
+    },
+    "value": 144,
+    "observedAt": "2022-01-05T23:00:00Z",
+    "unitCode": "SEC"
+  },
+  "windAverage1mn": {

```

Comment:
a bit weird to have the `1mn` in the property name. The property name could be `windAverage` and `1mn` a sub-property of the property.